### ISSUE TITLE
Build optimizations?

### ISSUE THREAD

**Issue by f0rttnite** [libsm64/libsm64-blender#17]
I'm assuming the addon works by building collision based off of mesh faces.
Would there be any room to optimize and speed it up?
I have a pretty decent cpu (ryzen 5 2600x) and even on scenes with 9,000 tris it takes a good minute or so for Blender to unfreeze

**Comment by jaburns** [comment]
Yeah there hasn't been much time spent on optimization of anything yet.
